variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/VariantClassifierTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.core.models;
+
+import org.junit.Rule;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import org.junit.rules.ExpectedException;
+
+public class VariantClassifierTest {
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testSNVs() {
+        assertEquals(VariantType.SNV, VariantClassifier.getVariantClassification("A", "C", 1));
+    }
+
+    @Test
+    public void testDIVs() {
```

Comment:
This grouping only makes sense following dbSNP rules, I think it is better if split in 3 test cases, one per EVA type.